src/aiida/storage/sqlite_dos/backend.py

```diff
@@ -45,7 +50,19 @@
 FILENAME_CONTAINER = 'container'
 
 
+ALEMBIC_REL_PATH = 'migrations'
+
+REPOSITORY_UUID_KEY = 'repository|uuid'
+
+
 class SqliteDosMigrator(PsqlDosMigrator):
+    """Class for validating and migrating `sqlite_dos` storage instances.
+
+    .. important:: This class should only be accessed via the storage backend class (apart from for test purposes)
+    """
+
+    alembic_version_tbl_name = 'alembic_version'
+
     """Storage implementation using Sqlite database and disk-objectstore container.
 
     This storage backend is not recommended for use in production. The sqlite database is not the most performant and it
```

Comment:
I am confused, it looks like this docstring should not be here.

Reply:
Indeed, this belongs under the `SqliteDosStorage` class which already has an updated docstring. So will just remove this.